config/application.yml.example

```diff
@@ -152,9 +152,18 @@ development:
   proofer_mock_fallback: 'true'
   rack_mini_profiler: 'off'
   reauthn_window: '120'
-  recaptcha_enabled_percent: '0'
-  recaptcha_site_key: 'key1'
-  recaptcha_secret_key: 'key2'
+  # reCAPTCHA global setting
+  recaptcha_enabled: 'false'
+  # Action key format: controller_path + '#' + action_name
+  # For example: '{ "sign_up/registrations#new": "true" }'
+  recaptcha_enabled_actions: '{
```

Comment:
I'm not a fan of making this on a per action basis.  You might want recaptcha on a per packet or per session basis (ie force recaptcha because you have detected nefarious activity).  As a minor note the json could get abnormally large and seems to muddy up the config file.

Reply:
I think we will want per-endpoint configuration, and in fact it's required since this PR lays the groundwork for the next one, which would see reCAPTCHA enabled for one feature, but not necessarily for others.

Reply:
By naming the actions we can support per-endpoint configurations but also support the other instances which I mention.  It also makes the config a little cleaner.